.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,104 @@
+name: "Release - Create PR to trunk"
+
+# This action will run inside the release code freeze workflow or when it is triggered manually
+on:
+  workflow_call:
+    inputs:
+      tagName:
+        description: 'The tag name (e.g. 4.5.0)'
+        required: true
+        type: string
+      releaseDateString:
+        description: "The release date in human-readable format, which will be formatted for changelog.txt and readme.txt (default: 'next wednesday')."
+        required: false
+        default: "next wednesday"
+        type: string
+  workflow_dispatch:
+    inputs:
+      tagName:
+        description: 'The tag name (e.g. 4.5.0)'
+        required: true
+        type: string
+      releaseDateString:
+        description: "The release date in human-readable format, which will be formatted for changelog.txt and readme.txt (default: 'next wednesday')."
+        required: false
+        default: "next wednesday"
+        type: string
+  
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  prepare-release:
+    name: Prepare a stable release
+    runs-on: ubuntu-20.04
+    outputs: 
+      branch: ${{ steps.create_branch_tag.outputs.branch-name }}
+    env:
+      TAG_NAME: ${{ inputs.tagName }}
+      RELEASE_DATE_STRING: ${{ inputs.releaseDateString }}    
+    
+    steps:
+      - name: Checkout WCPay repository
+        uses: actions/checkout@v3
+#        with:
```

Comment:
Can we remove these commented lines if not using them?

Reply:
I removed this part. We want the release-pr workflow to either work from the `develop` branch (the default one) if running inside the code freeze workflow or from the branch selected when manually triggered (I can imagine cases where we might want to create the release branch from a previous test branch if we're not comfortable including risky changes merged in week 3 for instance... Not too sure about this, but it doesn't hurt to leave that possibility).